### ISSUE TITLE
Accessibility review 

### ISSUE THREAD

**Issue by yochannah** [open-life-science/open-life-science.github.io#846]
See https://github.com/open-life-science/open-life-science.github.io/issues/438#issue-1403303617 for a suggestion on how to assess accessibility. We'll probably end up making several more issues as a result.

**Comment by SaraVilla** [comment]
- w3c web assessibility initiative](https://www.w3.org/WAI/)
- w3c WAI report tool](https://www.w3.org/WAI/eval/report-tool/)
- w3c WAI Web Content Accessibility Guidelines 2.2](https://www.w3.org/WAI/WCAG22/quickref/)
- Wave accessibility checking extension (for Firefox, Chrome-like)](https://wave.webaim.org/extension/)
- Record a Goose sighting](https://record-a-goose-sighting.apps.live.cloud-platform.service.justice.gov.uk/) (a fictional government service filled with accessibility problems for you to explore with Wave)
- #disability-and-accessibility in RSE Slack with a monthly meetup
- 24 hours of inclusive design broadcast](https://inclusivedesign24.org/)
